src/components/doc-footer/doc-footer.scss

```diff
@@ -1,14 +1,12 @@
-@import "../../styles/typography";
-@import "../../styles/content-spacing";
-@import "../../styles/global-css-settings";
+@import "../../styles/mixins/typography";
 
 .vibe-sb-comps-footer {
   &_text {
-    @include content-text;
+    @include sb-content-text;
```

Comment:
I didn't like the prefix sb for the mixins, I think it a bit redundant (the name is longer but it's not helping the mixin name to be more clear)

Reply:
(in the tokens I understand why it's needed)

Reply:
But here it's also serves same purpose, no? So you won't mixup docs-related and components-related mixins.
I know that technically you can import only the storybook mixin and it would work, but I think it gives more clear separation of concepts and makes it harder to make a mistake. Clear rule - if writing story or storybook component, use only tokens and mixins with `sb-` prefix

Reply:
mmmm I still think it's redundant but it's not a blocker